### PR TITLE
Update packages and remove redundant imports

### PR DESCRIPTION
**Pull request by JunTaoLuo**
#140

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
review: cc @javiercn 
approval: @Eilon

File: samples/MusicStore.Spa/Startup.cs [hunk 1]
```diff
@@ -54,14 +54,26 @@ public void ConfigureServices(IServiceCollection services)
...
+                cfg.CreateMap<Album, AlbumChangeDto>();
+                cfg.CreateMap<Album, AlbumResultDto>();
+                cfg.CreateMap<AlbumResultDto, Album>();
+                cfg.CreateMap<Artist, ArtistResultDto>();
+                cfg.CreateMap<ArtistResultDto, Artist>();
+                cfg.CreateMap<Genre, GenreResultDto>();
+                cfg.CreateMap<GenreResultDto, Genre>();
+            });
+
+			//Mapper.CreateMap<AlbumChangeDto, Album>();
```
Comment (JunTaoLuo) [inline samples/MusicStore.Spa/Startup.cs]: oops, will remove this

**Comment by javiercn** [comment]
Looks good to me

**Comment by javiercn** [comment]
:shipit:

**Pull request merged**